src/manipulation/wrapMap.js

```diff
@@ -8,19 +8,15 @@ var wrapMap = {
 
 	thead: [ 1, "<table>", "</table>" ],
 
-	// Some of the following wrappers are not fully defined, because
-	// their parent elements (except for "table" element) could be omitted
-	// since browser parsers are smart enough to auto-insert them
-
 	// Support: Android 2.3
```

Comment:
This comment should be removed as it suggests we're only putting `<colgroup>` here because of Android 2.3 and it's XML as well. The XHTML-related comment from below should be moved above this line.